AN00209_xCORE-200_DSP_Library/app_atan2_hypot/Makefile

```diff
@@ -1,22 +1,32 @@
+BUILD_FLAGS = -O2 -g
+
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
```

Comment:
Same comment as app_adaptive Makefile.

Reply:
This has been fixed by https://github.com/xmos/lib_dsp/pull/193